src/parser/main.js

```diff
@@ -0,0 +1,47 @@
+// TO RUN FILE: remove "type": "module" from package.json
+// otherwise will receive Error [ERR_REQUIRE_ESM]
+const fs = require('fs');
+const parse = require('./parseGeoID');
+const quant = require('./mergeQuantiles');
+const quantiles = require('../store/quant06085.json');
+
+function buildDataset(dir, geoInput) {
+
+    const scenarios = fs.readdirSync(dir)
+        .filter(file => file !== '.DS_Store');
+        //.slice(0,1); // shorten for testing
+    const severities = ['high', 'med', 'low'];
```

Comment:
If you will be using these values a lot, you can make a const with these values

Example:
In a constants file somewhere
```
export const SEVERITY = {
    HIGH: "high",
    MED: "med",
    LOW: "low"
}
```

To use
```
const severities = [SEVERITY.HIGH, SEVERITY.MED, SEVERITY.LOW];
```
or 
```
Object.values(SEVERITY)
```

It lets you refactor easily by making the hardcoded string only exist in one place as well as ensure that you are using the correct value and didn't typo.